README.md

```diff
@@ -81,69 +81,116 @@ Summary of one-time project setup:
+   - Remove the lines that contain `@pytest.mark.skip()` for the test(s) you want to run.
 
-3. Run the tests for your specific wave
+3. Run the test(s)! (RED)
+    - *See the [Details About How to Run Tests](#details-about-how-to-run-tests) section below for more information on how to run test(s).*
 
 ```bash
-# Must be in activated virtual environment
-$ pytest tests/test_wave_01.py
+# Must be in activated virtual environment in the project-root directory
+$ pytest
 ```
 
-4. Focus on the top test failure. Read through the test failure, and understand why the failure is happening. Confirm your findings with a classmate.
+4. Read through the test failure, and understand why the failure is happening. Confirm your findings with a classmate. 
+    - If it is a test you wrote, consider whether you are actually testing what you intend to test. Does the test need modification?
+
+5. Make a plan to implement code to pass the test.
 
-5. Make a plan to fix the test failure.
+6. Write code to pass the test (GREEN).
 
-6. Write code to fix the test failure.
+7. Re-run the test(s).
 
-7. Re-run the tests.
+8. Repeat steps 3-7 until that test passes!
 
-8. Repeat steps 5-7 until that test passes!
+9. Consider looking for opportunities to improve your code (REFACTOR)
 
-9. Repeats steps 4-8 until you have finished all tests in the file.
+10. Repeats steps 2-8 until you have finished all tests in the file.
 
-10. Begin using the test file of the next wave!
+11. Begin using the test file of the next wave!
 
-11. When you are finished working for the day, deactivate your environment with deactivate or closing the Terminal tab/window
+12. When you are finished working for the day, deactivate your environment with deactivate or closing the Terminal tab/window
 
 ```bash
 $ deactivate
 ```
+Finally, at submission time, **no matter where you are**, submit the project via Learn.
+
+This will let us give feedback on what you've finished so that you can be better prepared for the next project.
+
+## Integration Tests
+
+We provided integration tests for this project. Integration tests verify that individual units of a piece of software are working together correctly.
+
+The integration tests for this project are in the directory `tests/integration_tests`, and the tests have the decorator `@pytest.mark.integration_test` provided by the `pytest-integration` package. Marking these tests as integration tests makes them run after the unit tests. The isolated nature and specficity of *unit tests* make them a much better tool for debugging. Hence, we want to run the unit tests before the integration tests.
+
+The integration tests provided in this project do not cover all the test cases verified by the unit tests. They verify that the functions work together when invoked correctly. We could classify them as the subset of Integration Tests called *Smoke Tests*, tests that check basic functionality of the application.
```

Comment:
This paragraph really gets at what I meant with my prior comment on line 37. Would it make sense to move this up as part of that first paragraph that introduces what the integration tests are?

Reply:
Good call. I took a stab at reorganizing.

Reply:
I think I'm stuck on the sentence "Integration tests verify that individual units of a piece of software are working together correctly." To me, it doesn't add as much clarity as I would like.  

I was thinking something that compared them directly to unit tests like:
> While unit tests are small, fast, and should cover most of our program's behavior down to individual functions, integration tests verify that the various pieces of a program are working together correctly.

Reply:
I like Kelsey's suggestion. I think specifically, using the word 'units' in the definition of integration tests might undermine the contrast we're trying to draw between unit and integration tests.

Reply:
great! I incorporated this.